CMakeLists.txt

```diff
@@ -20,22 +19,18 @@ if(DEFINED ENV{CETBUILDTOOLS_VERSION})
 
     cet_report_compiler_flags()
 
-    find_ups_product( eigen )
-    find_ups_product( protobuf )
-    find_ups_product( tensorflow )
+    find_package(Protobuf REQUIRED)
+    find_package (Eigen3 3.3 REQUIRED NO_MODULE)
```

Comment:
Could you please confirm whether the exact version specification `3.3` is required here?

Reply:
I believe that 3.3 is specified because larsoft, and our build of tensorflow, specifically use 3.3.9.  LArSoft is unable to use a newer release of eigen until problems with the tensorflow build are resolved.

Reply:
Note: you should be able to use the CMake version range notation here, if you wish:
```CMake
find_package(Eigen 3.3...3.3.9 REQUIRED NO_MODULE)
```